alpha/apps/kaltura/lib/model/objectfilters/kBaseUserEntryAdvancedFilter.php

```diff
@@ -108,11 +108,6 @@ public function applyCondition(IKalturaDbQuery $query)
 
 			while (true)
 			{
-				if (count($entryIds) >= $limit)

```

Comment:
why we needed to remove the limit in general? looks like we will run on many more entries now